src/components/PortfolioAccordions.vue

```diff
@@ -38,46 +37,47 @@
         </div>
       </div>
 
-      <!-- Use the accurate heading level to maintain the document outline -->
-      <h4 class="usa-accordion__heading">
+
+     <!-- Use the accurate heading level to maintain the document outline -->
+      <h3 class="usa-accordion__heading">
         <button
           class="usa-accordion__button"
           aria-expanded="false"
```

Comment:
This and the accordion for 'Events' are marked as not expanded initially (`aria-expanded = "true"`), but they *are* expanded on page load. So for events, for example, it is expanded, but it thinks it isn't, so to close it you have to click to expand it (which makes no change) and then again to collapse it. Should we instead set both to `aria-expanded = "true"`, or were you intending for these to *not* be expanded on page load?

Reply:
Good catch, I wasn't sure what was doing that